bokehjs/src/lib/core/kinds.ts

```diff
@@ -526,6 +527,64 @@ export namespace Kinds {
     }
   }
 
+  export class ColorSpecValue extends Kind<types.ColorSpecValue> {
```

Comment:
Not to be confused with the Glyphs ColorSpecs, which are quite different from these. I think this one is more like the Python ColorSpec? The names are confusing

Reply:
AFAIK the existing `ColorSpec` should also be what it wanted here? i.e. it is a data spec that is specialized to accept scalar color values, or refer to color values from a CDS column. What issues did you run into trying to use the existing `ColorSpec`? 

cc @mattpap for any thoughts

Reply:
@bryevdv I couldn't manage to get them working at first, still unfamiliar with JS and the bokeh codebase, but I managed to use the default specs in latest commit